I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 524, in script
    Jogador "Consequências piores do que deixar o barco afundar? Vamos descobrir."
Exception: Sayer 'Jogador' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 524, in script
    Jogador "Consequências piores do que deixar o barco afundar? Vamos descobrir."
  File "C:\Users\Wilker Ribeiro\Downloads\renpy-8.2.2-sdk\renpy\ast.py", line 2560, in execute
    Say.execute(self)
  File "C:\Users\Wilker Ribeiro\Downloads\renpy-8.2.2-sdk\renpy\ast.py", line 575, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Wilker Ribeiro\Downloads\renpy-8.2.2-sdk\renpy\ast.py", line 464, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'Jogador' is not defined.

Windows-10-10.0.22621 AMD64
Ren'Py 8.2.2.24061201
projetoX 1.0
Thu Aug 22 21:33:04 2024
